wellcomeml/datasets/winer.py

```diff
@@ -216,8 +216,8 @@ def _load_data_spacy(data_path, inc_outside=True, merge_entities=True):
     X = []
     Y = []
     sentence_text = None
+    char_i = 0 # A counter to populate the start and end character indexes for each entity
```

Comment:
char_i is not a good name, hence the comment. ent_start_index could be a better one